src/frontends/tensorflow_common/src/op/ones_like.cpp

```diff
@@ -21,6 +21,11 @@ namespace op {
 OutputVector translate_ones_like_op(const NodeContext& node) {
     default_op_checks(node, 1, {"OnesLike"});
     auto x = node.get_input(0);
+    auto complex_type_mark_x = as_type_ptr<ComplexTypeMark>(x.get_node_shared_ptr());
+    if (complex_type_mark_x) {
+        x = complex_type_mark_x->input_value(0);
```

Comment:
Not correct implementation:)
That is because we receive ones in imaginary part but imag part should consists of zeros. Please implement separate logic for complex type.

![image](https://github.com/openvinotoolkit/openvino/assets/35459624/a0028035-f51c-4a71-81df-15892859c130)

Reply:
@rkazants thanks for your input. I'll amend the changes